.travis.yml

```diff
@@ -3,26 +3,34 @@ sudo: false
 branches:
   only:
     - master
-python:
-  - 3.5
-  - 3.6
-
-env: 
```

Comment:
I think you will need `python` and `env` back to [build the testing matrix](https://docs.travis-ci.com/user/build-matrix/)

Reply:
hmmm. yeah, [I tried that](https://github.com/pysal/spaghetti/pull/261/commits/754c05a671d69c2e61069e8b0aa297ac1421b64e#diff-354f30a63fb0907d4ad57269548329e3R7) and it broke the build due to the [`xenial` issue](https://github.com/travis-ci/travis-ci/issues/9815). I am wondering if we should simply skip either the `include` or `allow_failures` builds, because they each of them works well individually, but I can't seem to get them to work together at once.

@ljwolf Do you have any advice on this?

Reply:
try this https://github.com/weikang9009/spaghetti/blob/python37/.travis.yml

Reply:
try this https://github.com/weikang9009/spaghetti/blob/master/.travis.yml

Reply:
it is working now https://travis-ci.org/weikang9009/spaghetti

Reply:
I changed xenial as a global var

Reply:
Excellent! I will make that change!

Reply:
jobs on python 3.7 in allow_failures did not build because the jobs did not exactly match definitions given in allow_failures due to the global `sudo` and `dist` settings https://docs.travis-ci.com/user/build-matrix/#matching-jobs-with-allow_failures